### ISSUE TITLE
RSRS - PhyloTree 17.1 => Job failed

### ISSUE THREAD

**Issue by niki0211** [genepi/haplogrep3#29]
When choosing PhyloTree 17.1 (RSRS) as phylogenetic tree, I always get an error: "Job failed."
PhyloTree 17.0 works fine.

Input format: FASTA

**Comment by lukfor** [comment]
Thanks for reporting this! Any chance to share the input fasta file with us?

**Comment by niki0211** [comment]
Yes, I tried it with the reference sequence of homo sapiens neanderthalensis.
>NC_011137.1

**Comment by seppinho** [comment]
Thanks for reporting. It'S related to the new cluster annotation. RSRS 17.0 will give the same classification result.

**Comment by chanchaldevnani** [comment]
I have been finding the same error of Job failed, or unsupported file format.

Am uploading FASTA alignment both with and without refseq

**Comment by chanchaldevnani** [comment]
I have been facing the same error. Job failed, file format unsupported.

I am using FASTA Alignment.

**Comment by haansi** [comment]
HI, which phylogenetic tree did you use? For RSRS and neanderthal sequences please use the version 17.0 not the 17.1.
Do your fasta files sequences contain special characters (e.g. ":" or "-" ), does the file name end with ".fasta" or ".fa" (.txt will not work)? Try to get rid of "." in the filename unless as for the file extension seperator. 
No need to include a reference though - is your data from human samples?
